docs/api/migration-guides/qiskit-1.0-packaging.mdx

```diff
@@ -0,0 +1,646 @@
+</Admonition>
+
+You do not need to read this entire page to start using Qiskit 1.0.
+
+If you are a user, read the [For users](#for-users) section for the getting started guide.
+If you develop or maintain a package that depends on Qiskit, read the [For developers](#for-developers) section for recommendations on managing the transition.
+There is [a lot of detail on what exactly is changing and why it had to change](#why-did-this-happen) that you need only read if you are interested.
+There is a large [Troubleshooting section](#troubleshooting) which you can consult if you encounter problems installing or importing Qiskit 1.0.
+
+
+<span id="for-users"></span>
+## For users
+
+You must start a new virtual environment to install Qiskit 1.0.
+It is very tricky and error-prone to upgrade an existing installation in-place to Qiskit 1.0.
+
+<Admonition type="info">
+This section will use the `venv` module that is part of the Python standard library in its examples.
+If you use a different tool (such as `virtualenv` or `conda`), you should consult its documentation if you are unsure of how the commands line up.
+
+For Linux and macOS commands, we will use a bash-like syntax for the command-line syntax.
+For Windows we will use PowerShell.
+</Admonition>
+
+<span id="creating-a-venv"></span>
+### Creating the new environment
+
+1. Create a new virtual environment, using your preferred version of Python 3.8 or higher:
+
+   Linux and macOS:
+
+   ```bash
+   python3 -m venv /path/to/qiskit-1.0-venv
+   ```
+
+   Windows with PowerShell:
+
+   ```powershell
+   python3 -m venv C:\path\to\qiskit-1.0-venv
+   ```
+
+   You can use any path you like in place of `/path/to/qiskit-1.0-venv`.
+
+2. Activate the environment.
+
+   Linux and macOS:
+
+   ```bash
+   source /path/to/qiskit-1.0-venv/bin/activate
+   ```
+
+   Windows with PowerShell:
+
+   ```powershell
+   C:\path\to\qiskit-1.0-venv\Scripts\activate.ps1
+   ```
+
+3. Install all the packages you want to install.
+   You should do this using only one `pip install` command with all the dependencies on it.
+
+   ```bash
+   pip install 'qiskit>=1'
+   ```
+
+   You can include other packages you want installed alongside `qiskit` here by writing them as following arguments, such as
+
+   ```bash
+   pip install 'qiskit>=1' jupyterlab pandas matplotlib
+   ```
+
+   <Admonition type="tip">
+   Qiskit 1.0 includes breaking changes, and several packages will have marked themselves as not-yet-compatible with it, so you might see errors from `pip` when running this until they have new versions.
+   Old versions of packages may also depend on the legacy `qiskit-terra` package, which might not immediately error out during this command, but might raise an error when running `import qiskit`.
+
+   If you know of any packages that are not yet compatible with Qiskit 1.0, you will have to not install those yet, and wait until the packages are updated.
```

Comment:
Perhaps here you can link to the section on how to use `pipdeptree` to find dependency versions.

Reply:
I put in an extra comment and link in f4b16f09, though I avoided dropping an actual link to the specific troubleshooting section below, because that's all about recovering from a broken environment.  I don't actually know a simple way to get `pip` (or `pipdeptree`) to tell you the requirements of a particular prospective package without attempting to install it (though maybe the experimental `pip index` will eventually do that).